src/components/Blocking/ReportPostAnonymous.js

```diff
@@ -108,7 +109,7 @@ const styles = StyleSheet.create({
   title: {
     fontFamily: fonts.inter[700],
     fontSize: 16,
-    color: '#000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Nice work replacing the hardcoded color with a value from the theme. This enhances code readability and maintainability.
```diff
- color: '#000',
+ color: COLORS.black,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->